src/LayerEditor/ui/data/layer_model.py

```diff
@@ -1,8 +1,13 @@
+from collections import deque
+
+from bgem.external import undo
```

Comment:
`better_undo` doesn't provide everything from `undo`, its not a full wrapper. `better_undo` was more of a hotfix, since we are not getting `undo` from pip it might be better to just integrate it to undo. But I will let you decide :)

Reply:
in better_undo: `from bgem.external.undo import *`